addon/instance-initializers/flexberry-logging.js

```diff
@@ -10,6 +10,16 @@ import Ember from 'ember';
   @for FlexberryLoggingService
  */
 export function initialize(applicationInstance) {
+
+  /**
+      flexberry-logging service for transmit error/warning/log/info/debug/deprecation messages to  store and save its on server as i-i-s-caseberry-logging-objects-application-log object
```

Comment:
If you leave this comment (I do not understand what for) then there are too many spaces at the beginning of line.